File: app/src/main/java/fr/nihilus/xenobladechronicles/di/MainActivityModule.java
package fr.nihilus.xenobladechronicles.di;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;
import fr.nihilus.xenobladechronicles.MainActivity;

@Module
public abstract class MainActivityModule {
    @ContributesAndroidInjector(modules = {FragmentBuildersModule.class})
    abstract MainActivity contributeMainActivity();
}

File: app/src/main/java/fr/nihilus/xenobladechronicles/di/MainActivityModule.kt
package fr.nihilus.xenobladechronicles.di

import dagger.Module
import dagger.android.ContributesAndroidInjector
import fr.nihilus.xenobladechronicles.MainActivity

@Module
abstract class MainActivityModule {
    @ContributesAndroidInjector(modules = [FragmentBuildersModule::class])
    abstract fun contributeMainActivity(): MainActivity
}
